### PR TITLE
Add countries API resource

### PR DESCRIPTION
**Pull request by andrewisplinghoff**
This add the documentation of the new countries API resource from the PR https://github.com/shopware/shopware/pull/524 to the developer guide website.

### CODE REVIEW

File: source/developers-guide/rest-api/api-resource-countries/index.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+---
+layout: default
+title: REST API - Countries Resource
+github_link: developers-guide/rest-api/api-resource-countries/index.md
+indexed: false
```
Comment (bcremer) [inline source/developers-guide/rest-api/api-resource-countries/index.md]: Can you please add:
`shopware_version: 5.3`

Comment (andrewisplinghoff) [inline source/developers-guide/rest-api/api-resource-countries/index.md]: As discussed with @MarcelSchmaeing, it should be included in 5.2. Should I add `shopware_version: 5.2` instead?

Comment (bcremer) [inline source/developers-guide/rest-api/api-resource-countries/index.md]: Oh yeah, that was a typo on my side.

Comment (andrewisplinghoff) [inline source/developers-guide/rest-api/api-resource-countries/index.md]: OK, done https://github.com/shopware/devdocs/pull/284/commits/7a49807b441d67d89f9e864b547d37ce89a287e7

**Pull request merged**